gems.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+gem 'activesupport'
+gem 'httparty'
```

Comment:
I don't think `httparty` is needed on current level of code, it's used for making HTTP request - and currently I don't see one